anaconda.spec.in

```diff
@@ -58,7 +58,6 @@ BuildRequires: make
 BuildRequires: pango-devel
 BuildRequires: python3-kickstart >= %{pykickstartver}
```

Comment:
I think that this used to be a dependency of the nose tests, so we could remove it as well.

Reply:
All tests seem to pass with it locally, let's see.